server/migrations/20220427145338-alterAtVersionTable.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.addColumn(
+                'AtVersion',
+                'id',
+                {
+                    type: Sequelize.DataTypes.INTEGER,
+                    allowNull: false,
+                    primaryKey: true,
+                    autoIncrement: true
+                },
+                { transaction }
+            );
+            await queryInterface.addColumn(
+                'AtVersion',
+                'releasedAt',
+                {
+                    type: Sequelize.DataTypes.DATE
+                },
+                { transaction }
```

Comment:
I feel like this might be a good migration to add a value for releasedAt, making sure that column is fully populated. When it's populated it will be possible to make the field required. As for what value to use, I don't think it matters much. It can be today's date. In a future migration in the TestPlanTarget PR I'm going to delete all the currently existing versions anyway.